app/controllers/api/v1/cars_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V1::CarsController < ApplicationController
+  def index
+    @cars = Car.all
+    render json: @cars
+  end
+
+  def show
+    @car = Car.find(params[:id])
+    render json: @car
+  end
+
+  def create
+    @car = Car.new(car_params)
+    if @car.save
+      render json: @car, status: :created
+    else
+      render json: @car.errors, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @car = Car.find(params[:id])
+    @car.destroy
+    head :no_content
```

Comment:
- It would be better if you can show the user a more user-friendly response when deleting a non-existing car model.

![error](https://user-images.githubusercontent.com/88746054/229733384-754efea6-5abd-4ce4-b1fb-fe3586cdda67.png)

- You can achieve this by changing `find` to `find_by(id: params[:id])` and checking the returned value with an if statement
```ruby
    @car = Car.find_by(id: params[:id])
    if @car&.destroy
     # delete car
    else
      # return a user friendly message
    end
```